common/eslint-jest.config.mjs

```diff
@@ -0,0 +1,38 @@
+import eslintTsConfig from './eslint-ts.config.mjs';
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+
+const tsEslint = require('typescript-eslint');
+const jest = require('eslint-plugin-jest');
+
```

Comment:
Interesting.  What was the reason for this?

Reply:
The dependencies are packaged as commonjs, not ES modules.